staging/src/k8s.io/legacy-cloud-providers/azure/azure_managedDiskController.go

```diff
@@ -229,6 +229,10 @@ func (c *ManagedDiskController) DeleteManagedDisk(diskURI string) error {
 
 	disk, rerr := c.common.cloud.DisksClient.Get(ctx, resourceGroup, diskName)
 	if rerr != nil {
+		if rerr.HTTPStatusCode == http.StatusNotFound {
```

Comment:
I see on line 245 we already check for http.StatusNotFound when we call Delete, does that already handle this?

Reply:
Oh I see now, we need to call Get because of the ManagedBy check on line 239.